src/community/ogcapi/web-features/src/main/java/org/geoserver/ogcapi/web/FeatureServiceAdminPanel.html

```diff
@@ -0,0 +1,254 @@
+<html xmlns:wicket="http://wicket.apache.org/">
+<head></head>
+<body>
+<wicket:panel>
+    <h3><wicket:message key="defaultTitle">Feature Service</wicket:message></h3>
+    <p><wicket:message key="configureInformation">Feature Service is a modular OGC API service with "building blocks" that can be enabled and configured for use.</wicket:message></p>
+    <ul>
+        <li>
+            <fieldset>
+                <legend><span><wicket:message key="features">OGC API - Features</wicket:message></span></legend>
```

Comment:
I did not expect to see this page hand-written... actually it might be a problem.
What if a given conformance class is provided through a plugin?

Probably better to have enough configuration to auto-generate this from the conformance classes that might be found in the spring context.

Reply:
I am walking before I can run on this one Andrea. I think the next step is to make a conformance panel that can be used multiple times. 

Want to get something that functions at all here to control the module.

Reply:
If a conformance class is provided through a plugin it provides its own `AdminPagePanel`, as that is the existing UI infrastructure for contributing.

Reply:
That is one possibility, but it's work-intensive and clumsy. Alsor remember admin panels are not following an order, you can have panels out of any logical order, and other unrelated extensions have their eventual panel  show up in the middle.

Reply:
I am thinking how to do that, and we have an extension that provides:
- APIConformance being configured
- property expression `metadata/cql2/text` from WFSInfo

Then FeatureServiceAdminPanel could do the very simple story we have now. But not additional settings.

May be better to have an extension like ConformancePanel, with a default implementation for the simple case.